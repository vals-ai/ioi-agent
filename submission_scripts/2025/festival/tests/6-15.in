version https://git-lfs.github.com/spec/v1
oid sha256:e7b2a7700f243838209571b732b822ed4d5106275beb6ebf010a2bcd771f0b46
size 1200051
