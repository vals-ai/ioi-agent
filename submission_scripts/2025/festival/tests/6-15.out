version https://git-lfs.github.com/spec/v1
oid sha256:ce53938e008318f2aa69413c1b6f73becdb8f31148ad333aaec8985a49b69b33
size 212
